### DISCUSSION TITLE
Parcel inlining of html templates

### DISCUSSION THREAD

**Discussion by fis-cz** [parcel-bundler/parcel#8585] [Q&A]
Hi,

I would like to import HTML template as string. But the template does not fit w3c as well as styles in it does not fit the correct format as there are placeholders for bindings.

when I try:

```typescript
import "bundle-text:./template.html"
```

while the template looks like:

```html
<template>
   <div style="{style}">Test</div>
</template>
```

I get 

```
🚨 [parcel]: @parcel/transformer-css: Unexpected token CurlyBracketBlock
D:\VSCode\Projects\TypeScript\ViewModelTest\src\components\Test\template.html:1:1
> 1 | {style}
>   | ^
```

Its clear that this is not vaild and parsable HTML/CSS code but its processed later.

Is there a way how to import whatever as string without running transformations over it?

Solution with fs mentioned doc does not work for me as I am bundling web app.

Thanks

**Comment by mischnic** [comment]
Either you use [readFileSync](https://parceljs.org/features/node-emulation/#inlining-fs.readfilesync),

or add a .parcelrc and then use `bundle-text-raw:./template.html` instead:
```json
{
    "extends": "@parcel/config-default",
    "transformers": {
        "bundle-text-raw:*": ["@parcel/transformer-inline-string"]
    }
}
```

**Reply by fis-cz** [reply]
Second solutions is great, thanks. The first one would require reference to the lib what I don't want.

Thanks!
